mmv1/products/spanner/InstancePartition.yaml

```diff
@@ -88,25 +89,33 @@ properties:
   - name: 'nodeCount'
     description: |
       The number of nodes allocated to this instance partition. One node equals
-      1000 processing units. Exactly one of either node_count or processing_units
-      must be present.
+      1000 processing units. Exactly one of either node_count, processing_units, or
+      autoscaling_config must be present.
     type: Integer
     validation:
       function: 'validation.IntAtLeast(1)'
-    exactly_one_of:
+    at_least_one_of:
```

Comment:
why not just stay with exactly_one_of and add the new field? That would be functionally equivalent and easier to understand.